test/runtime/Policies.go

```diff
@@ -687,12 +687,21 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 	BeforeEach(func() {
 		once.Do(initialize)
 		vm.PolicyDelAll()
+
+		vm.SampleContainersActions(helpers.Create, helpers.CiliumDockerNetwork)
+		areEndpointsReady := vm.WaitEndpointsReady()
```

Comment:
Whenever we instantiate these containers, I think we want to know that they're ready when it's returned. It'd be nice to fold in these waits (and maybe even the `Expect()`) into the `SampleContainersActions()` function. Maybe by using `gomega.ExpectWithOffset(...)` in the helper function so that ginkgo points out the correct line in the actual test file where it failed.

Reply:
This is a good idea, but in the interest of making sure that the specified tests are testing with endpoints running, I'm going to merge this.